hcl/attribute.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Mineiros GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hcl
+
+import "github.com/hashicorp/hcl/v2/hclsyntax"
+
+// Attribute represents a parsed attribute.
+type Attribute struct {
+	origin string
```

Comment:
Origin info will be good when we eval, so we can provide accurate data about where the original attribute came from.